src/comm/client/client.js

```diff
@@ -90,14 +92,43 @@ class Client{
         let conf = require(config);
         this.config = conf.test.clients;
         this.results = [];                        // output of recent test round
-        this.updates = {id:0, data:[]};           // contains txUpdated messages
+        this.updates = {id:0, data:[]}; // contains txUpdated messages
     }
 
     /**
     * Initialise client object
+    * @param {JSON} demo object which trakcs real time metrics
+    * @param {String} config path of the configuration file
+    * @param {String} absCaliperDir caliper directory
+    * @param {JSON} listener_child child process
+    * @param {JSON} t test object
     * @return {Promise} promise object
     */
-    init() {
+    init(demo, config, absCaliperDir, listener_child) {
+        absConfigFile = require(Util.resolvePath(config));
```

Comment:
since absConfigFile can be recorded using config framework, it is not necessary to add config as a parameter. 
You can use config framework like this:  const cfUtil = require('./config-util.js');  const cfUtil.getConfigSetting('fabric:WITH_MQ',false);   
If there is any problem when using this config framework, please let me know or you can contact with @feihujiang